src/components/about/about.svg

```diff
@@ -0,0 +1,20 @@
+<svg width="249" height="350" viewBox="0 0 249 350" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Comment:
I am happy to move these SVGs to the assets folder I created. I am happy for this PR to be merged if you are :)